.eslintrc.base.js

```diff
@@ -35,5 +35,11 @@ module.exports = {
         jest: true,
       },
     },
+    {
+      files: ['src/index.ts'],
+      rules: {
+        'node/shebang': 0,
+      },
+    },
```

Comment:
In general, this rule exists to help with cross platform compat. 🤔

Reply:
See the description in #43 for why the rule was working improperly (tldr; it had no clue `src/index.ts` was known as `dist/index.ts` when published).